### PR TITLE
EBean gradle plugin - make compatible with EBean 10.x and suitable for java projects

### PR DESCRIPTION
**Pull request by vnnv**
Hi Rob, 
I try to fix the plugin to be compatible with latest Ebean. For me it works. Could you please review the changes and if you are ok publish it to be available in maven central.

### CODE REVIEW

**Comment by rbygrave** [comment]
This PR can be closed now.  I've bumped the agent up to 10.2.x which has combined entity bean and query bean enhancement etc.

**Comment by vnnv** [comment]
Hi Rob, could you please update Example build.gradle because it seems outdated. Also if I use my current build.gradle with the latest plugin - enhancement is not started.

**Comment by vnnv** [comment]
Hy Rob, I made some additional changes - 
1. rename the packages to io.ebean
2. fix Gradle 4.0 warning - output directoryes for every compiled language
3. fix: ebean.mf is no considered because compileJava task is invoked before processResources

File: src/main/groovy/io/ebean/gradle/EbeanEnhancer.groovy [hunk 1]
```diff
@@ -54,6 +52,7 @@ class EbeanEnhancer {
         classInputStream.close()
 
         String jvmClassName = className.replace('.','/')
+        logger.info("Try to enhance $jvmClassName");
```
Comment (rbygrave) [inline src/main/groovy/io/ebean/gradle/EbeanEnhancer.groovy:55]: Do we really want this at info level?  I might drop this down to debug afterwards.

**Comment by rbygrave** [comment]
Good stuff, merging - thanks !!!

File: src/main/groovy/io/ebean/gradle/EbeanEnhancer.groovy (hunk 1 above)
Comment (vnnv) [reply src/main/groovy/io/ebean/gradle/EbeanEnhancer.groovy:55]: no actually :)

**Pull request merged**